modules/scan/port.yaml

```diff
@@ -1027,6 +1027,9 @@ payloads:
         response:
           condition_type: or
           conditions:
+            exec:
```

Comment:
not sure if there was a better hack.

Reply:
we should not be using 'exec' to call external tools like openssl. There are  **ssl** **cryptography** and **pyopenssl**  libraries available built-into Python which can run natively

Reply:
Unfortunately pyopenssl seems to not support all the functionality I needed.